src/main/scala/mesosphere/marathon/core/matcher/reconcile/impl/OfferMatcherReconciler.scala

```diff
@@ -52,7 +52,10 @@ private[reconcile] class OfferMatcherReconciler(taskTracker: TaskTracker, groupR
           case (taskId, spuriousResources) if spurious(taskId) =>
             val unreserveAndDestroy =
               TaskOp.UnreserveAndDestroyVolumes(
-                taskId = taskId, oldTask = tasksByApp.task(taskId), resources = spuriousResources.to[Seq]
+                // FIXME (3221): is ForceExpunge correct here?
+                newTask = TaskStateOp.ForceExpunge(taskId),
```

Comment:
yes, for now `ForceExpunge` should be correct. Once we really use `Garbage` states and similar things, it will no be correct.